app/models/katello/subscription.rb

```diff
@@ -13,8 +13,9 @@ class Subscription < Katello::Model
     scope :in_organization, ->(org) { where(:organization => org) }
 
     def self.subscribable
+      product_ids = Product.subscribable.pluck(:id) + [nil]
```

Comment:
Ok, now why is this `+ [nil]` here now? `pluck` already returns an array.

Reply:
[nil] is required for the query below, we want it to return items where the value is either the list of those product ids or nil